release-notes/release-notes-insiders.json

```diff
@@ -1,4 +1,31 @@
 [
+    {
+        "version": "1.6.1-insiders",
+        "title": "Fiddler Everywhere v1.6.1",
+        "date": "May 27, 2021",
+        "items": [
+            {
+                "type": "Fixed",
+                "entries": [
+                    {
+                        "title": "Unable to set proxy on macOS when adapter's name has multiple spaces next to each other"
+                    },
+                    {
+                        "title": "Unable to set proxy on macOS's default Thunderbolt adapters"
+                    },
+                    {
+                        "title": "System proxy is not respected when starting the application on macOS in some cases"
+                    },
+                    {
+                        "title": "The system proxy is not used for some of the initial requests during application startup"
+                    },
+                    {
+                        "title": "Skip multiple tracking of same analytics events when application starts"
```

Comment:
Text should be something less scary for users - e.g. analytics update, or optimized analytics during startup.

Reply:
Perhaps something like this will sound more friendly:

_Improved analytics behavior during application startup._

or

_Optimized analytics events execution during application startup._

Reply:
I'll just remove it :D